src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -225,6 +227,10 @@ abstract class FixupSession(
 
   private fun showAcceptGroup() {
     showLensGroup(LensGroupFactory(this).createAcceptGroup())
+
+    // This is the specific moment after which any edits to the document,
+    // including edits generated by the Agent, will result in an auto-accept.
+    documentListener.setAcceptLensGroupShown(true)
```

Comment:
not sure if it'd help but maybe we could add a check to verify if the task.state is `CodyTaskState.Applied` before we add the listener? Not a blocker though!